core/mm/mobj.c

```diff
@@ -164,24 +190,7 @@ struct mobj *mobj_phys_alloc(paddr_t pa, size_t size, uint32_t cattr,
 		return NULL;
 	}
 
-	/* Only SDP memory may not have a virtual address */
```

Comment:
Did you drop this part by mistake or on purpose?

Reply:
It was accidentally lost on refactoring. Fixed, thanks!